docs/maps/search.asciidoc

```diff
@@ -49,13 +49,15 @@ Spatial filters have the following properties:
 * *Geometry label* enables you to provide a meaningful name for your spatial filter.
 * *Spatial field* specifies the geo_point or geo_shape field used to determine if a document matches the spatial relation with the specified geometry.
 * *Spatial relation* determines the {ref}/query-dsl-geo-shape-query.html#_spatial_relations[spatial relation operator] to use at search time. Only available when *Spatial field* is set to geo_shape.
+* *Action* specifies where the filter will be applied. Select *Apply filter to current view* to apply the filter to the current view. Select a drilldown action to apply the filter to the drilldown action. Only available when the map is a panel in a {kibana-ref}/dashboard.html[dashboard] with {kibana-ref}/drilldowns.html[drilldowns].
 
 [float]
 [[maps-phrase-filter]]
 ==== Phrase filters
 
 A phrase filter narrows search results to documents that contain the specified text.
 You can create a phrase filter by clicking the plus icon image:maps/images/gs_plus_icon.png[] in a <<maps-vector-tooltip-locking, locked tooltip>>.
+When the map is a panel in a {kibana-ref}/dashboard.html[dashboard] with {kibana-ref}/drilldowns.html[drilldowns], you can apply a phrase filter to a drilldown by selecting the drilldown action.
```

Comment:
I don't think you need to link to dashboard and drilldowns here as the links are in the preceding paragraph.

Also, maybe this:

If the map is a dashboard  panel with drilldowns, you can select the drilldown action _from the panel menu._ (or wherever the action is)